sci-physics/rivet/rivet-3.1.10.ebuild

```diff
@@ -93,6 +93,10 @@ src_configure() {
 		$(usex python CYTHON="${ESYSROOT}/usr/bin/cython")
 }
 
+src_test() {
+	emake check
+}
```

Comment:
Isn't this the default `src_test` already?

Reply:
I thought so too, but `pkg-testing-tools` does not detect the tests ( maybe portage api changed?) i.e.:

```python
>>> import portage
>>> portage.portdb.aux_get("sci-physics/yoda-1.9.10", ["DEFINED_PHASES"])
['configure install postinst prepare setup test']
>>> portage.portdb.aux_get("sci-physics/rivet-3.1.10", ["DEFINED_PHASES"])
['configure install prepare setup']
```

Reply:
`sudo ebuild rivet-3.1.10 test` however executes tests

Reply:
That the phase is not defined does not mean that it does not run. The default phase should always be run if the phase is not defined, and the default is documented here: https://devmanual.gentoo.org/ebuild-writing/functions/src_test/index.html

If this is not the case this is a bug `pkg-testing-tools`.

Reply:
Yes, I already asked for an alternative in the portage API https://bugs.gentoo.org/927946, Then I'll close this and update the other rivet PR.